README.rst

```diff
@@ -86,6 +86,21 @@ Registry
               to:
                 - name@receivehost.com
 
+Docker login to private registry
+--------------------------------
+
+.. code-block:: yaml
+
+    docker:
+      host:
+        enabled: true
+        registries:
```

Comment:
It would be better as dict imo:
```
host:
  registry:
    private.docker.com:
      user: test
      password: test
```
So you will be able to override some parameters easily in reclass.